source/extensions/filters/http/ratelimit/ratelimit.cc

```diff
@@ -126,9 +127,11 @@ void Filter::onDestroy() {
 }
 
 void Filter::complete(Filters::Common::RateLimit::LimitStatus status,
-                      Http::HeaderMapPtr&& headers) {
+                      Http::HeaderMapPtr&& response_headers,
+                      Http::HeaderMapPtr&& request_headers_to_add) {
   state_ = State::Complete;
-  headers_to_add_ = std::move(headers);
+  response_headers_to_add_ = std::move(response_headers);
```

Comment:
I think the naming is still a bit inconsistent; `response_headers` should be actual response headers (coming from the upstream), `response_headers_to_add` should be used (just like `request_headers_to_add`) to refer to the configured response additions.

Reply:
Modified